src/OpenTelemetry.Exporter.Zipkin/OpenTelemetryBuilderExtensions.cs

```diff
@@ -0,0 +1,90 @@
+﻿// <copyright file="OpenTelemetryBuilderExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using OpenTelemetry.Exporter.Zipkin;
+
+namespace OpenTelemetry.Trace.Configuration
+{
+    /// <summary>
+    /// Extension methods to simplify registering of Zipkin exporter.
+    /// </summary>
+    public static class OpenTelemetryBuilderExtensions
+    {
+        /// <summary>
+        /// Registers a Zipkin exporter that will receive <see cref="System.Diagnostics.Activity"/> instances.
+        /// </summary>
+        /// <param name="builder"><see cref="OpenTelemetryBuilder"/> builder to use.</param>
+        /// <param name="configure">Exporter configuration options.</param>
+        /// <returns>The instance of <see cref="OpenTelemetryBuilder"/> to chain the calls.</returns>
+        public static OpenTelemetryBuilder UseZipkinActivityExporter(this OpenTelemetryBuilder builder, Action<ZipkinTraceExporterOptions> configure)
+        {
+            if (builder == null)
+            {
+                throw new ArgumentNullException(nameof(builder));
+            }
+
+            if (configure == null)
+            {
+                throw new ArgumentNullException(nameof(configure));
+            }
+
+            return builder.AddProcessorPipeline(pipeline =>
+            {
+                var options = new ZipkinTraceExporterOptions();
+                configure(options);
+
+                var activityExporter = new ZipkinActivityExporter(options);
+                pipeline.SetExporter(activityExporter);
+            });
+        }
+
+        /// <summary>
+        /// Registers a Zipkin exporter that will receive <see cref="System.Diagnostics.Activity"/> instances.
+        /// </summary>
+        /// <param name="builder"><see cref="OpenTelemetryBuilder"/> builder to use.</param>
+        /// <param name="configure">Exporter configuration options.</param>
+        /// <param name="processorConfigure">Activity processor configuration.</param>
+        /// <returns>The instance of <see cref="OpenTelemetryBuilder"/> to chain the calls.</returns>
+        public static OpenTelemetryBuilder UseZipkinActivityExporter(this OpenTelemetryBuilder builder, Action<ZipkinTraceExporterOptions> configure, Action<ActivityProcessorPipelineBuilder> processorConfigure)
```

Comment:
Why do we need a callback to configure the `ActivityProcessorPipelineBuilder` here? Wouldn't that be on the builder itself (one level up)? Doesn't really have anything to do with Zipkin.
```csharp
Action<ActivityProcessorPipelineBuilder> processorConfigure
```
Did you mean it to be `Action<ZipkinActivityExporter> configureExporter`?

Reply:
This overload allows one to use Zipkin exporter, along with own customization to the processor pipeline. Like add MyProcessor, and SetExporter would be done with
builder.UseZipKinActiivtyExporter(configureZipkinOptions, (pipeline) => pipeline.Add(MyProcessor));

Without this overload, there is no easy way to add MyProcessor to the same pipeline as Zipkin is attached to.

One could still do 
builder.AddProcessorPipeline( (pipeline) => pipeline.Add(MyProcessor).SetExporter(new ZipkinExporter())) though. The above is letting user continue to use the Zipkin extension method.

Reply:
I'll wait to hear from @CodeBlanch on this comment before merging this PR.

Reply:
Got it! Sounds good to me. Although @rajkumar-rangaraj we really don't need two methods. Just make both params default to `null` eg:

```csharp
public static OpenTelemetryBuilder UseZipkinActivityExporter(this OpenTelemetryBuilder builder, Action<ZipkinTraceExporterOptions> configure = null, Action<ActivityProcessorPipelineBuilder> processorConfigure = null)
{
            if (builder == null)
            {
                throw new ArgumentNullException(nameof(builder));
            }

            return builder.AddProcessorPipeline(pipeline =>
            {
                var options = new ZipkinTraceExporterOptions();
                configure?.Invoke(options);

                var activityExporter = new ZipkinActivityExporter(options);
                pipeline.SetExporter(activityExporter);
                processorConfigure?.Invoke(pipeline);
            });
}

Reply:
Yes this is even better.

Reply:
Fixed